apps/meteor/app/cors/server/cors.ts

```diff
@@ -86,6 +99,37 @@ WebAppInternals.staticFilesMiddleware = function (
 	next: NextFunction,
 ) {
 	res.setHeader('Access-Control-Allow-Origin', '*');
+	const { arch, path, url } = WebApp.categorizeRequest(req);
+
+	if (
+		Meteor.isProduction &&
+		!cacheControlledRoutes.some((route) => {
+			return new RegExp(`^${route}`, 'gi').test(path);
+		})
```

Comment:
complementing the suggestion above:
```suggestion
		!cacheControlledRoutes.some((regexp) => regexp.test(path))
```